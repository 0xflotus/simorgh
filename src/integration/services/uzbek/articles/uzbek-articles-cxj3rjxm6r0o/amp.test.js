/**
 * @service uzbek
 * @pathname /uzbek/articles/cxj3rjxm6r0o
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP uzbek articles', runAmpTests);
