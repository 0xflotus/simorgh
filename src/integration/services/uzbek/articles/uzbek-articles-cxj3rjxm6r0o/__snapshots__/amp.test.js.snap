// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP uzbek articles A11y I can see the skip to content link 1`] = `"Саҳифага ўтиш"`;

exports[`AMP uzbek articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP uzbek articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP uzbek articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/uzbek"`;

exports[`AMP uzbek articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. Би-би-си ташқи сайтлар мазмуни учун масъул эмас. Ташқи линкларга бизнинг ёндашувимиз қандайлиги ҳақида маълумотга эга бўлинг"`;

exports[`AMP uzbek articles Header I can see the branding 1`] = `"BBC News, O'zbek"`;

exports[`AMP uzbek articles SEO Canonical link 1`] = `"http://localhost:7080/uzbek/articles/cxj3rjxm6r0o"`;

exports[`AMP uzbek articles SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP uzbek articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP uzbek articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP uzbek articles SEO Heading level 1 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи?"`;

exports[`AMP uzbek articles SEO Lang attribute 1`] = `"uz-cyrl"`;

exports[`AMP uzbek articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
      "width": 1024,
    },
    "name": "BBC News O'zbek",
    "noBylinesPolicy": "https://www.bbc.com/uzbek/institutional-50220995#authorexpertise",
  },
  "dateModified": "2019-07-12T14:34:24.846Z",
  "datePublished": "2019-07-12T14:34:24.846Z",
  "headline": "Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи?",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/uzbek/articles/cxj3rjxm6r0o",
    "@type": "WebPage",
    "name": "Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи?",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
      "width": 1024,
    },
    "name": "BBC News O'zbek",
    "publishingPrinciples": "https://www.bbc.com/uzbek/institutional-50220995",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png",
  "url": "http://localhost:7080/uzbek/articles/cxj3rjxm6r0o",
}
`;

exports[`AMP uzbek articles SEO OG description 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи?"`;

exports[`AMP uzbek articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png"`;

exports[`AMP uzbek articles SEO OG image alt 1`] = `"BBC News O'zbek"`;

exports[`AMP uzbek articles SEO OG locale 1`] = `"uz-UZ"`;

exports[`AMP uzbek articles SEO OG site name 1`] = `"BBC News O'zbek"`;

exports[`AMP uzbek articles SEO OG title 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи? - BBC News O'zbek"`;

exports[`AMP uzbek articles SEO OG type 1`] = `"article"`;

exports[`AMP uzbek articles SEO OG url 1`] = `"http://localhost:7080/uzbek/articles/cxj3rjxm6r0o"`;

exports[`AMP uzbek articles SEO Page title 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи? - BBC News O'zbek"`;

exports[`AMP uzbek articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP uzbek articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP uzbek articles SEO Twitter creator 1`] = `"@bbcuzbek"`;

exports[`AMP uzbek articles SEO Twitter description 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи?"`;

exports[`AMP uzbek articles SEO Twitter image alt 1`] = `"BBC News O'zbek"`;

exports[`AMP uzbek articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/uzbek.png"`;

exports[`AMP uzbek articles SEO Twitter site 1`] = `"@bbcuzbek"`;

exports[`AMP uzbek articles SEO Twitter title 1`] = `"Ўзбекистон: «Энергетик таъмли ичимлик» нима ўзи? - BBC News O'zbek"`;
